api/controllers/HomeController.js

```
@@ -28,7 +28,7 @@ module.exports = {
           }
         });
       }
-    });
+    }, sails.log.error);
```

When the user loads the index page, we refresh their flair, but only after sending a response. This was causing errors to get swallowed.